assets/javascripts/admintable.js

```diff
@@ -31,7 +35,9 @@ function showAdminTableRow(row) {
     }
 
     // scroll to the new row
-    $('html').animate({scrollTop: $(row.node()).offset().top}, 250);
+    $('html').animate({
+        scrollTop: $(row.node()).offset().top
+    }, 250);
```

Comment:
The previous style looked better.

Reply:
see https://github.com/os-autoinst/openQA/pull/3268#discussion_r456940506